packages/js/test/plugins/auctionHouseModule/directBuy.test.ts

```diff
@@ -64,6 +63,101 @@ test('[auctionHouseModule] buy on an Auction House with minimum input', async (t
   } as unknown as Specifications<Purchase>);
 });
 
+test('[auctionHouseModule] partial buy on an Auction House', async (t: Test) => {
+  // Given we have an Auction House and an NFT.
+  const mx = await metaplex();
+  const seller = await createWallet(mx);
+  const auctionHouse = await createAuctionHouse(mx);
+  const sft = await createSft(mx, {
+    tokenOwner: seller.publicKey,
+    tokenAmount: token(10)
+  });
+
+  await mx.tokens().mint({
+    mintAddress: sft.address,
+    amount: token(10),
+    toOwner: seller.publicKey
+  });
```

Comment:
I don't think this is required since you're already setting `tokenAmount: token(10)` when creating the SFT.

```suggestion
```